umberto.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
I'm thinking... maybe it would be cleaner if this file was kept in `docs/`? There are already many files in the root dir of every package.